frontend/src/app/main/data/workspace/transforms.cljs

```diff
@@ -847,12 +847,20 @@
           (rx/of (reorder-selected-layout-child direction))
           (rx/of (nudge-selected-shapes direction shift?)))))))
 
+(defn- get-relative-delta [position bbox parent]
```

Comment:
I'll call the third parameter `frame`, because this not the direct parent, but the nearest frame (the direct parent may be a group, for example).

Reply:
Same for `parent-bbox` below